rotkehlchen/chain/ethereum/airdrops.py

```diff
@@ -143,12 +146,12 @@ def fetch_airdrops_metadata(database: 'DBHandler') -> tuple[dict[str, Airdrop],
     with GlobalDBHandler().conn.read_ctx() as cursor:
         airdrops_metadata_cached_at = globaldb_get_unique_cache_last_queried_ts_by_key(
             cursor=cursor,
-            key_parts=(CacheType.AIRDROPS_METADATA,),
+            key_parts=(CacheType.AIRDROPS_METADATA, AIRDROPS_INDEX_VERSION),
```

Comment:
as I said above shouldn't the index be a variable? Here you always use the hardcoded version. So if a new one exists you won't know and just fetch the one from the DB?